Classes/Finishers/EmailFinisher.php

```diff
@@ -89,6 +89,9 @@ protected function executeInternal()
         if ($recipientAddress === null) {
             throw new FinisherException('The option "recipientAddress" must be set for the EmailFinisher.', 1327060200);
         }
+        if (is_array($recipientAddress) && $recipientName !== '') {
+            throw new \TYPO3\Form\Exception\FinisherException('The option "recipientName" cannot be used with multiple recipients in the EmailFinisher.', 1483365977);
```

Comment:
&lt;nitpick&gt;No need for the fully qualified namespace, FinisherException is already imported&lt;nitpick&gt;